src/community/dds/src/main/java/org/geoserver/wms/worldwind/BilMapResponse.java

```diff
@@ -1,352 +1,372 @@
-                CRS.transform(destinationEnvelope, cvCRS);
-
-        /** Reading Coverage on Requested Envelope */
-        Rectangle destinationSize = null;
-        destinationSize = new Rectangle(0, 0, request.getHeight(), request.getWidth());
-
-        /** Checking for supported Interpolation Methods */
-        Interpolation interpolation = Interpolation.getInstance(Interpolation.INTERP_BILINEAR);
-
-        // /////////////////////////////////////////////////////////
-        //
-        // Reading the coverage
-        //
-        // /////////////////////////////////////////////////////////
-        Map<Object, Object> parameters = new HashMap<Object, Object>();
-        parameters.put(
-                AbstractGridFormat.READ_GRIDGEOMETRY2D.getName().toString(),
-                new GridGeometry2D(
-                        new GeneralGridEnvelope(destinationSize), destinationEnvelopeInSourceCRS));
-
-        final GridCoverage2D coverage =
-                coverageReader.read(
-                        CoverageUtils.getParameters(
-                                coverageReader.getFormat().getReadParameters(), parameters, true));
-
-        if (coverage == null) {
-            LOGGER.log(Level.FINE, "Failed to read coverage - continuing");
-            return null;
-        }
-
-        /** Band Select */
-        /*
-           Coverage bandSelectedCoverage = null;
-
-           bandSelectedCoverage = WCSUtils.bandSelect(request.getParameters(), coverage);
-        */
-        /** Crop */
-        final GridCoverage2D croppedGridCoverage =
-                BilWCSUtils.crop(
-                        coverage,
-                        (GeneralEnvelope) coverage.getEnvelope(),
-                        cvCRS,
-                        destinationEnvelopeInSourceCRS,
-                        Boolean.TRUE);
-
-        /** Scale/Resampling (if necessary) */
-        // GridCoverage2D subCoverage = null;
-        GridCoverage2D subCoverage = croppedGridCoverage;
-        final GeneralGridEnvelope newGridrange = new GeneralGridEnvelope(destinationSize);
-
-        subCoverage =
-                BilWCSUtils.scale(
-                        croppedGridCoverage,
-                        newGridrange,
-                        croppedGridCoverage,
-                        cvCRS,
-                        destinationEnvelopeInSourceCRS);
-
-        /** Reproject */
-        subCoverage = BilWCSUtils.reproject(subCoverage, sourceCRS, targetCRS, interpolation);
-
-        return subCoverage;
-    }
-
-    /** This is not really an image map */
-    @Override
-    public MapProducerCapabilities getCapabilities(String outputFormat) {
-        // FIXME become more capable
-        return new MapProducerCapabilities(false, false, false, false, null);
-    }
-
-    static {
-        RecodeRaster.register(JAI.getDefaultInstance());
-    }
-}
+/* (c) 2014-2015 Open Source Geospatial Foundation - all rights reserved
```

Comment:
This is a full reformat, it is not admissible (actual changes are hidden) and won't pass the Travis QA tests. Please build the module with Maven, it will automatically format it with the expected conventions.

Reply:
> This is a full reformat, it is not admissible (actual changes are hidden) and won't pass the Travis QA tests. Please build the module with Maven, it will automatically format it with the expected conventions.

I have fixed the format.

Reply:
This is still showing as every line changed.

Reply:
> This is still showing as every line changed.

It's strange that I have built the extension with Maven.You see it passed the Travis test.I just add some lines.
I'm not familiar with java and I will try to rebuild the entire project instead of building the dds extension.